test/suite/test_import01.py

```diff
@@ -0,0 +1,188 @@
+    scenarios = make_scenarios([
+        # Importing into a new database.
+        ('new', dict(import_type='new')),
+        # Importing into an existing database with other files.
+        ('existing', dict(import_type='existing')),
+        # Importing into the same database. We should expect a failure.
+        ('same', dict(import_type='same')),
+    ])
+
+    def update(self, uri, key, value, commit_ts):
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction()
+        cursor[key] = value
+        self.session.commit_transaction('commit_timestamp=' + timestamp_str(commit_ts))
+        cursor.close()
+
+    def check(self, uri, key, value, read_ts):
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction('read_timestamp=' + timestamp_str(read_ts))
+        cursor.set_key(key)
+        self.assertEqual(0, cursor.search())
+        self.assertEqual(value, cursor.get_value())
+        self.session.rollback_transaction()
+        cursor.close()
+
+    # Helper for populating a database to simulate importing files into an existing database.
+    def populate(self):
+        # Create file:test_import01_[1-100].
+        for fileno in range(1, 100):
+            uri = 'file:test_import01_{}'.format(fileno)
+            self.session.create(uri, 'key_format=i,value_format=S')
+            cursor = self.session.open_cursor(uri)
+            # Insert keys [1-100] with value 'foo'.
+            for key in range(1, 100):
+                cursor[key] = 'foo'
+            cursor.close()
+
+    def copy_file(self, file_name, old_dir, new_dir):
+        if os.path.isfile(file_name) and "WiredTiger.lock" not in file_name and \
+            "Tmplog" not in file_name and "Preplog" not in file_name:
+            shutil.copy(os.path.join(old_dir, file_name), new_dir)
+
+    def test_file_import(self):
+        original_db_file = 'original_db_file'
+        uri = 'file:' + original_db_file
+
+        create_config = 'allocation_size=512,key_format=u,log=(enabled=true),value_format=u'
+        self.session.create(uri, create_config)
+
+        key1 = b'1'
+        key2 = b'2'
+        key3 = b'3'
+        key4 = b'4'
+        value1 = b'\x01\x02aaa\x03\x04'
+        value2 = b'\x01\x02bbb\x03\x04'
+        value3 = b'\x01\x02ccc\x03\x04'
+        value4 = b'\x01\x02ddd\x03\x04'
+
+        # Add some data.
+        self.update(uri, key1, value1, 10)
+        self.update(uri, key2, value2, 20)
+
+        # Perform a checkpoint.
+        self.session.checkpoint()
+
+        # Add more data.
+        self.update(uri, key3, value3, 30)
+        self.update(uri, key4, value4, 40)
+
+        # Perform a checkpoint.
+        self.session.checkpoint()
+
+        # Export the metadata for the table.
+        c = self.session.open_cursor('metadata:', None, None)
+        original_db_file_config = c[uri]
+        c.close()
+
+        self.printVerbose(3, '\nFILE CONFIG\n' + original_db_file_config)
```

Comment:
I think we can remove the debugging code? I missed it on commit.

Reply:
It's fine becuase I've switched it to `printVerbose` because this only appears if you run with `-v 3`.